packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ProxyEOA.sol

```diff
@@ -47,20 +38,16 @@ contract OVM_ProxyEOA {
     fallback()
         external
     {
-        (bool success, bytes memory returndata) = Lib_SafeExecutionManagerWrapper.safeDELEGATECALL(
-            gasleft(),
-            getImplementation(),
-            msg.data
-        );
+        (bool success, bytes memory returndata) = getImplementation().delegatecall(msg.data);
 
         if (success) {
             assembly {
                 return(add(returndata, 0x20), mload(returndata))
             }
```

Comment:
This success case doesn't seem to match the one in `_safeExecutionManagerInteraction`:
https://github.com/ethereum-optimism/optimism/blob/17654d06a034b655243d63a27ecbf21da290a255/packages/contracts/contracts/optimistic-ethereum/libraries/wrappers/Lib_SafeExecutionManagerWrapper.sol#L350-L360